ygen/protogen.go

```diff
@@ -134,8 +195,20 @@ func genProtoMsg(msg *yangDirectory, msgs map[string]*yangDirectory, state *genS
 	}
 
 	definedFieldNames := map[string]bool{}
+	imports := []string{}
+
+	// Traverse the fields in alphabetical order to ensure determinsitic output.
```

Comment:
deterministic

Reply:
Fixed elsewhere.

Reply:
I mean you have a typo there: s/determinsitic/deterministic/